### ISSUE TITLE
Several problems regarding waypoints/pathing and /npc move

### ISSUE THREAD

**Issue by charlie1990** [CitizensDev/Citizens#904]
I'm sorry for the giant wall of text, there is a TLDR version at the bottom.

This all started the other day when we started re-doing our spawn, we were doing heavy worldediting in the area and decided to move all of the npcs at our spawn(49) to a floating island we  made so they didn't get caught in the edits. After we were all done and ready to move them back the first problem came up, /npc move wasn't working or appeared to not  work, it looked like it didn't move but you could see their text when you went near the area we tried moving them to and if we tried teleporting to them it would put us at the location we tried moving them to but when we went to the island all of the npcs were still there, upon lots of testing i discovered that the /npc move command was working if it was kept to a short range(19 blocks to be exact) I thought it was weird and decided to just delete all of our npcs since we had so many unused ones anyways and all of the "new" npcs after the removal didn't have a problem with the command and moved just fine.

Onto problem two which arose today, we are about to open an adventure map and i've been tasked with setting the npcs up, this is going perfectly fine until i start setting the waypoints for the city guards that roam the upper walls of the city..as i'm setting the first guards waypoints i turn around and notice he's missing, I then teleport to him to find him walking through the city, and not in a normal way, he's actually turning corners, crossing bridges and avoiding hitting things, it was very strange but the point is he deviated completely off his waypoints, every npc i created after this suffered the same fate after about 3 minutes of walking along their correct waypoints. I eventually just let one roam off and followed it, it managed to walk through a forest extremely accurately(avoiding hitting trees, avoiding water etc) eventually it turned around and tried to walk back to it's home location but it walked  for a good 10 minutes before doing so. I told the server owner who then pointed out to me that an important quest npc was missing in the main world, i teleported to the npc and it was more then 3000 blocks away  from its waypoints...i moved it back to its correct location to see how it even got out, i watched it walk down 3 spiral staircases PERFECTLY without falling even though it had no waypoints near those staircases or outside of the building...

Problem Three, as I was equipping the city guards from problem two i noticed that i wasn't able to see any of the items equipped, the message displays saying it's equipped but i cant see it. No this is not a client  side bug, I have re-logged, left the area and came back, switched worlds and came back and it still didn't have it's equipped items visible.

TLDR: NPCs weren't accepting the /npc move command in heavily world-edited areas, NPCs are deviating from their assigned waypoints by more then 3000 blocks and equipped items are not showing up on NPCs
Citizens Version 1.1.6, Minecraft Version 1.1, Bukkit Version 1.1R4
We haven't installed any new plugins or updated any of our plugins in the last week either.

**Comment by charlie1990** [comment]
Problem 2 seems very similar if not the same as issue #881
